app/views/skins/_skin_style_block.html.erb

```diff
@@ -1,28 +1,4 @@
-<!-- expects local "skin" and is cached one level up -->
+<% # expects local "skin" and is cached one level up %>
 <style type="text/css">
```

Comment:
This partial is only used for work skins. The reset of the stuff had no reason to be there.